Content.Client/MouseRotator/MouseRotatorSystem.cs

```diff
@@ -28,18 +28,15 @@ public override void Update(float frameTime)
         if (player == null || !TryComp<MouseRotatorComponent>(player, out var rotator))
             return;
 
-        var xform = Transform(player.Value);
-
         // Get mouse loc and convert to angle based on player location
         var coords = _input.MouseScreenPosition;
         var mapPos = _eye.PixelToMap(coords);
 
         if (mapPos.MapId == MapId.Nullspace)
             return;
 
-        var angle = (mapPos.Position - xform.MapPosition.Position).ToWorldAngle();
-
-        var curRot = _transform.GetWorldRotation(xform);
+        var angle = (mapPos.Position - _transformSystem.GetMapCoordinates(player.Value).Position).ToWorldAngle();
+        var curRot = _transformSystem.GetWorldRotation(player.Value);
```

Comment:
pass in transform component here since you're now resolving it twice

Reply:
Just use GetWorldPositionRotation instead of the 2 separate ones as well as it'll be much faster.

Reply:
Ah, sure.